robottelo/ui/locators/base.py

```diff
@@ -1047,7 +1047,8 @@
     "users.current_password": (By.ID, "user_current_password"),
     "users.password": (By.ID, "user_password"),
     "users.password_confirmation": (By.ID, "user_password_confirmation"),
-    "users.user": (By.XPATH, "//a[contains(., '%s')]"),
```

Comment:
If the user tries to search for himself on users page - this locator will return 2 elements, the first one is username at top right corner (user's menu).